src/components/SignerPanel/SigningStatus.js

```diff
@@ -110,64 +107,85 @@ class SigningStatus extends React.Component {
     )
   }
   render() {
-    const { status } = this.props
+    const { theme, status } = this.props
     return (
-      <React.Fragment>
-        <Status>
+      <div>
+        <Status
+          color={theme.feedbackContent}
+          background={theme.feedbackSurface}
+        >
           <StatusImage status={status} />
-          <p>{this.getLabel()}</p>
+          <p
+            css={`
+              margin-top: ${3.5 * GU}px;
+              ${textStyle('body2')};
+            `}
+          >
+            {this.getLabel()}
+          </p>
         </Status>
-        <AdditionalInfo>{this.getInfo()}</AdditionalInfo>
+        <Info>{this.getInfo()}</Info>
         {this.getCloseButton()}
-      </React.Fragment>
+      </div>
     )
   }
 }
 
 const Status = styled.div`
-  margin-top: 80px;
-  margin-bottom: 40px;
+  background: ${({ background }) => background};
+  height: 360px;
+  margin-bottom: ${3 * GU}px;
   display: flex;
   flex-direction: column;
   justify-content: center;
   align-items: center;
-  color: ${theme.textSecondary};
-  img {
-    margin-bottom: 20px;
-  }
+  color: ${({ color }) => color};
 `
 
-const AdditionalInfo = styled(Info)`
-  p + p {
-    margin-top: 10px;
-  }
-`
+const StatusImage = ({ status }) => {
```

Comment:
Perhaps we can lift this as its own component, as we use it in various places? Perhaps `StatusIndicator` or `FeedbackStatus`?

Reply:
This is similar to `You`, these are so small that I don't know if `aragonUI` should expose them but, it does make sense to standardise sizing, text styling and coloring for these components - which means, bringing them into `aragonUI`.

Thoughts @bpierre ?

Reply:
It probably wouldn’t hurt to make it its own component in aragon/aragon to start with, removing the `isSignatureError()` etc. inside. And from there, we could check with @dizzypaty and decide if it should be made available to apps (=> aragonUI), or if it is specific to aragon/aragon and can stay there.

Reply:
Yes, it should become an aragonUI component but we need to spec it first, standarize its size size & when it's presented.

Reply:
Moved it into its own component in 80f8b79 and gave it a small transition for the colour.